ga4gh/backends.py

```diff
@@ -280,30 +280,30 @@ def __init__(self, variantSetId, vcfPath):
         # self.tb chrom -> vcf dict
         self._tabixMap = {}
 
-        for fn in glob.glob(os.path.join(vcfPath, "*.vcf.gz")):
-            tabixfile = pysam.Tabixfile(fn)
+        for vcfFile in glob.glob(os.path.join(vcfPath, "*.vcf.gz")):
+            tabixfile = pysam.Tabixfile(vcfFile)
             for chrom in tabixfile.contigs:
                 if chrom in self._tabixMap:
                     raise Exception("cannot have overlapping VCF files.")
                 self._tabixMap[chrom] = tabixfile
 
     def convertVariant(self, record):
-        v = protocol.GAVariant()
```

Comment:
I will leave project-wide style guides and such issues for a follow up PR. For a public project I think clarity should trump terseness at all times. In any case it is easier to understand code where variables say clearly what they are. This makes code more self-documenting, and decreases ramp-up time. For example, I had to re-read this code many times to figure out what to rename variables to, indicating that they were not clear to begin with.